configstack/module.go

```diff
@@ -536,9 +537,16 @@ func FindWhereWorkingDirIsIncluded(terragruntOptions *options.TerragruntOptions,
 	var matchedModulesMap = make(map[string]*TerraformModule)
 
 	gitTopLevelDir, err := shell.GitTopLevelDir(terragruntOptions, terragruntOptions.WorkingDir)
+	useIncludes := false
```

Comment:
If i read the code correctly, there is never a case where false is required as value. Instead of creating the if condition we could maybe rearrange the code to be more straightforward.

Line 541 only runs when line 539 doesn't produce an error. The error is not handled until bit later so you might want to do that instead and simply set `useIncludes := true`. Call `buildDirList` safely after and only override with `false` when no errors are present.

Actually, after reading the line, just put 549-557 into a function and call it. `useIncludes` might not be required at all.